index.js

```diff
@@ -36,10 +36,9 @@ module.exports = {
         },
         ( err, files ) => {
           files.sort((a, b) => {
-            const sameDir = path.dirname(a) == path.dirname(b)
-            if (sameDir && isReadme(a))
+            if (path.dirname(b).includes(path.dirname(a)) && isReadme(a))
```

Comment:
May I ask why did you remove the `const sameDir` declaration? Wouldn't have been easier to just patch that line with:

```js
const sameDir = path.dirname(b).includes(path.dirname(a))
```

Basically becomes a one-line patch.

Reply:
Ah true, sorry. I did miss the swap of the letters. Visually they looked like the same.

Allright, safe enough to try :) Thank you!